applications/teletype/priv/templates/voicemail_deleted.text

```diff
@@ -0,0 +1,38 @@
+                Voicemail Deleted
+
+
+Hi{% if user.first_name %} {{user.first_name}}{% endif %},
+
+A voicemail from {{caller_id.name_number}} for
+your voicemail box at {{voicemail.vmbox_name}} (number: {{voicemail.vmbox_number}}) was deleted.
+Please find the message audio file in the attachment.
+{% if voicemail.transcription %}
+
+Delete reason: {{reason}}
```

Comment:
Please change the label to more readable "Deletion Reason:" string, same thing for html template file.

Reply:
We will update for this.